utilities/chemiscope.py

```diff
@@ -66,7 +66,7 @@ def chemiscope_input(meta, frames, projection, prediction,
                      properties, property_names=None,
                      untrained_properties=None, untrained_property_names=None,
                      cutoff=None,
-                    ):
+                     ):
```

Comment:
Was this added by a formatter or manually?

Reply:
This was added by a formatter